src/components/List.jsx

```diff
@@ -29,6 +30,14 @@ export const List = ({ token }) => {
     });
   };
 
+  const deleteItem = async (document) => {
+    if (
+      window.confirm(`Are you sure you want to delete ${document.data().item}?`)
+    ) {
+      await deleteDoc(doc(db, token, document.id));
+    }
+  };
+
```

Comment:
I like that you included the specific item in the alert!

Reply:
thank you!